agent/app/service/app_install.go

```diff
@@ -746,22 +746,6 @@ func (a *AppInstallService) GetParams(id uint) (*response.AppConfig, error) {
 						}
 					}
 				}
-			} else if form.Type == "apps" {
-				if m, ok := form.Child.(map[string]interface{}); ok {
-					result := make(map[string]string)
-					for key, value := range m {
-						if strVal, ok := value.(string); ok {
-							result[key] = strVal
-						}
-					}
-					if envKey, ok := result["envKey"]; ok {
-						serviceName := envs[envKey]
-						if serviceName != nil {
-							appInstall, _ := appInstallRepo.GetFirst(appInstallRepo.WithServiceName(serviceName.(string)))
-							appParam.ShowValue = appInstall.Name
-						}
-					}
-				}
 			}
 
 			params = append(params, appParam)
```

Comment:
There are no known changes between the two versions of the code snippet provided. The function `GetParams` looks fine; it gets parameters specified through web forms. Please let me know if you need any help with another code piece or concept.